person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_survive
+    age >= 18 || parent_permission == true
+  end
+
+  private
+
+  def of_age
+    @age >= 18
+  end
```

Comment:
- [ ] [OPTIONAL] It is a code style convention to put a question mark behind a method if it returns a boolean value (true or false) or an object to indicate a true value (or “truthy” value).